packages/flame_texturepacker/test/new_format_test.dart

```diff
@@ -44,8 +44,8 @@ void main() {
       expect(jumpSprite.rotate, true);
       expect(jumpSprite.name, 'robot_jump');
       expect(jumpSprite.srcPosition, Vector2(196, 260));
-      expect(jumpSprite.packedHeight, 192);
-      expect(jumpSprite.packedWidth, 256);
+      expect(jumpSprite.packedHeight, 256);
+      expect(jumpSprite.packedWidth, 192);
```

Comment:
can we add a test using the new useOriginalSize option and showcasing the difference? just need to assert the size variables

Reply:
Sure, will add it at next commit